app/core/calculations/nightevents.py

```diff
@@ -35,8 +36,31 @@ class NightEvents:
     moonset: Time
 
     # Information for Julian years.
-    _JULIAN_BASIS: ClassVar[float] = 2451545.0
-    _JULIAN_YEAR_LENGTH: ClassVar[float] = 365.25
+    _JULIAN_BASIS: ClassVar[float] = field(default=2451545.0, init=False, repr=False, compare=False)
```

Comment:
I think this variables are in some form in lucupy.sky

Reply:
Thanks! I'll check and remove them if they are.